install.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+MAVEN_METADATA=$(curl -sL http://central.maven.org/maven2/org/arquillian/smart/testing/smart-testing-parent/maven-metadata.xml)
+LATEST=$(echo ${MAVEN_METADATA} | grep -oPm1 "(?<=<latest>)[^<]+")
+
+VERSION=${LATEST}
+INSTALL_SPECIFIC_VERSION="0"
+
+while test $# -gt 0; do
+       case "$1" in
+            -h|--help)
+                echo "Installs Arquillian Smart Testing Extension"
+                echo "options:"
+                echo "-l, --latest                   installs latest version of the extension (default)"
+                echo "-v, --version=VERSION          installs defined version (doesn't check if exists!)"
+                exit 0
+                ;;
+            -l|--latest)
+                shift
+                VERSION=${LATEST}
+                shift
+                ;;
+            -v)
+                shift
+                if test $# -gt 0; then
+                    VERSION=$1
+                    INSTALL_SPECIFIC_VERSION="1"
+                else
+                    echo "No version specified."
+                    exit 1
+                fi
+                shift
+                ;;
+            --version*)
+                VERSION=`echo $1 | sed -e 's/^[^=]*=//g'`
+                INSTALL_SPECIFIC_VERSION="1"
+                shift
+                ;;
+            *)
+               echo "$1 is not a recognized flag!"
+               exit -1
+               ;;
+      esac
+done
+
+function install_shaded_library() {
+    if [ -z "$M2_HOME" ]; then
+        echo "Please set M2_HOME pointing to your Maven installation."
+        exit 1
+    fi
+    SHADED_JAR="maven-lifecycle-extension-${VERSION}-shaded.jar"
+    echo "Installing ${SHADED_JAR} into ${M2_HOME}/lib/ext"
+    wget http://central.maven.org/maven2/org/arquillian/smart/testing/maven-lifecycle-extension/${VERSION}/${SHADED_JAR}
+    read -r -p "We want to move shaded jar to M2_HOME with sudo. Can we? [y/N] " response
+    case "$response" in
+        [yY][eE][sS]|[yY])
+              sudo mv $SHADED_JAR $M2_HOME/lib/ext
+            ;;
+        *)
+            ;;
+    esac
+}
+
+function install_extension() {
+
+    # Needs to be without new lines, otherwise my sed skills below will fail badly :\
+    EXTENSION="<extension><groupId>org.arquillian.smart.testing</groupId><artifactId>maven-lifecycle-extension</artifactId><version>${VERSION}</version></extension>"
```

Comment:
simple:
`EXTENSION="<extension>\\n<groupId>org.arquillian.smart.testing</groupId>\\n<artifactId>maven-lifecycle-extension</artifactId>\\n<version>${VERSION}</version>\\n</extension>"` should work I think.

then, the xmllint could be used for the final formatting only if it is available...

Reply:
I would like to leave it like it is with the lint

Reply:
I meant visual new lines in the editor not `\n` BTW